server/views/partials/footer.html

```diff
@@ -0,0 +1,26 @@
+<footer class="govuk-footer " role="contentinfo">
```

Comment:
is there a reason why we cant use the govuk footer macro with this?

Reply:
Was initially following the html format of header and layout already in place but seems like no reason not to use the nunjucks macro - implemented as suggested